stories/__tests__/SQFormDialog.stories.test.js

```diff
@@ -168,3 +168,40 @@ describe('Tests for WithAutoFocus', () => {
     expect(textfield).toBe(document.activeElement);
   });
 });
+
+describe('Tests for Tertiary Button', () => {
+  it('should display the tertiary button', async () => {
+    const tertiaryButtonText = 'Tertiary Button';
+    render(
+      <Default
+        isOpen={true}
+        onSave={handleSave}
+        onClose={handleClose}
+        showTertiaryButton={true}
+        tertiaryButtonText={tertiaryButtonText}
+        title="Test"
+      />
+    );
+
+    await screen.findByText(tertiaryButtonText);
```

Comment:
There are no expects in this test, so nothing is actually being tested. Probably want to add a check for if the button is visible and maybe with the expected button text.

Reply:
@laurelbean I added the tests, but if you could double-check them I would appreciate it.  Pretty new to writing tests...

Reply:
It's looking better!

The 2 lines below are essentially doing the same thing, checking for the existence of the button. So you don't need both of them. Also, unless you're expecting to wait for loading or an animation, you don't need to `waitFor` an expect. 

```
expect(
   screen.getByRole('button', {name: tertiaryButtonText})
).toBeInTheDocument();

await waitFor(() => {
    expect(screen.getByText(tertiaryButtonText)).toBeInTheDocument();
});```

If you wanted to check for the text of the button, then I'd save the `getByRole` to a variable, and check that for existence and for text. Something like:
```
const tertiaryButton = screen.getByRole('button', {name: tertiaryButtonText});
expect(tertiaryButton).toBeInTheDocument();

expect(tertiaryButton).toHaveTextContext(tertiaryButtonText);
or 
expect(tertiaryButton).toHaveText(tertiaryButtonText);
```

Reply:
@laurelbean Thanks for helping me out with this!  The correct changes should now be part of this PR.  Let me know if anything further needs to be addressed.